frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -27,10 +27,8 @@ interface SubmissionFilterProps {
 /* This helper ensures start range values are inclusive
  * of the day set in the date picker. */
 const inclusiveDateString = (originalDate: string) => {
-    let inclusiveDateDate = new Date(originalDate);
-    return new Date(
-        inclusiveDateDate.setUTCHours(23, 59, 59, 999)
-    ).toISOString();
+    let inclusiveDate = new Date(originalDate);
```

Comment:
nit: doesn't need to be `let`, can be `const`

Reply:
Agree with Etan here about using `const` instead.

Alternatively, we don't even need to assign a variable since it's such a short-lived reference 🤷 
```tsx
return new Date(
    new Date(originalDate).setUTCHours(23, 59, 59, 999)
).toISOString();
```